### ISSUE TITLE
As a Platform, I want to Deploy an OG API Record to GitHub and provide automated schema validation, so that I can ingest a valid OGC API Record into the EarthCODE Catalogue

### ISSUE THREAD

**Issue by GarinSmith** [ESA-EarthCODE/EarthCODE#32]
Hi @Schpidi, given that  Vito (@JanssenBrm ) have stated their current preference is to publish Products and Experiments directly to GitHub, can we please move this story to ToDo? This is because this is now a dependency for Vito and anybody else who wishes to publish Products and Experiments directly to GitHub.
Thanks

**Comment by Schpidi** [comment]
see current status at https://github.com/ESA-EarthCODE/EarthCODE/issues/25#issuecomment-2605744135

**Comment by Schpidi** [comment]
latest discussion regarding validation is happening at https://github.com/ESA-EarthCODE/open-science-catalog-validation/issues/37

**Comment by Schpidi** [comment]
Agreed to close this story as soon as we have an initial example and schema (see item 2 in https://github.com/ESA-EarthCODE/open-science-catalog-validation/issues/37

**Comment by Schpidi** [comment]
A meeting with EOEPCA+ was held on 27.8.2025 and next steps agreed towards a joint approach for collaborative schema development.
